composer.json

```diff
@@ -59,5 +60,6 @@
         "branch-alias": {
             "dev-master": "0.3-dev"
         }
-    }
+    },
+    "minimum-stability": "dev"
```

Comment:
does cakephp have other unreleased dependencies? if not, afaik the version constraint for cakephp should be enough to make composer install a dev version.

Reply:
this is only needed for now
This PR is WIP while cake5 is beta